src/index.html

```diff
@@ -2,16 +2,165 @@
+          </label>
+          <input 
+            type="checkbox" 
+            name="trems" 
+            id="terms-and-conditions" 
+            class="field"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input 
+            type="email" 
+            placeholder="email@example.com" 
+            name="email" 
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="6" 
+            maxlength="20" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="no"
+            >
+            No
+          </label>
+        </label>
+
+        <label class="form_field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color"
+          >
+        </label>
+
+        <label class="form_field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="time" 
+            step="2"
+          >
+        </label>
+
+        <label class="form_field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form_field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="range" 
+            min="1" 
+            max="5" 
+            value="-1"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label  class="form_field">
+          Comments:
+          <textarea 
+            name="comments" 
+            cols="20" 
+            rows="2"
+          >
+          </textarea>
+        </label>
+
+        <label class="form_field">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="yes" selected>yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+      <button type="submit">Submit</button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+</html>
```

Comment:
Don't forget to add a new line at the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
ok